clusterloader2/run-e2e.sh

```diff
@@ -22,5 +22,10 @@ CLUSTERLOADER_ROOT=$(dirname "${BASH_SOURCE}")
 export KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
 export KUBEMARK_ROOT_KUBECONFIG="${KUBEMARK_ROOT_KUBECONFIG:-${HOME}/.kube/config}"
 
+# Deploy a secret that contains the e2e Google credentials
+if [[ -z "${E2E_GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
```

Comment:
log/echo a warning if this couldn't be found. Looks like if the file doesn't exist we just give up